src/tools/interop/idt/discovery/matter_ble.py

```diff
@@ -0,0 +1,108 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import asyncio
+import datetime
+import logging
+import os
+import time
+
+from bleak import AdvertisementData, BleakScanner, BLEDevice
+from bleak.exc import BleakDBusError
+
+
+class MatterBleScanner:
```

Comment:
`hcidump -Xt -w` has been useful to me.

Reply:
`dbus-monitor --system`` has been useful

Reply:
`btmon` has been usefule

Reply:
How do you feel about adding bt to `capture`?